app/logic/Calendar/Event.ts

```diff
@@ -162,17 +162,16 @@ export class Event extends Observable {
     this.duration = days * 86400;
   }
 
-  /** Create a new instance of the same event.
-   * Copy all data of the `original` event into this new Event object */
+  /**
+   * Create a new instance of the same event.
+   * Copy most data of the `original` event into this new Event object.
```

Comment:
It must copy *all* data.
If you need a variation, you can pass in optional parameters. Please make sure to document when you would use which variant.